src/lib/packager/clients/repositories.rb

```diff
@@ -1252,9 +1248,7 @@ def SummaryDialog
                 end
                 schema = Builtins.tolower(Builtins.substring(url2, 0, 3))
 
-                if schema != "cd:" && schema != "dvd"
-                  to_refresh = Ops.add(to_refresh, 1)
-                end
+                to_refresh = Ops.add(to_refresh, 1) if schema != "cd:" && schema != "dvd"
```

Comment:
```suggestion
                to_refresh += 1 if schema != "cd:" && schema != "dvd"
```

Reply:
btw is it not a typo that "cd:" and "dvd" without colon?

Reply:
This is quite tricky, it's the result of
```ruby
schema = Builtins.tolower(Builtins.substring(url2, 0, 3))
```

So for "dvd:///" it is just "dvd", but for "cd:///" it is "cd:" including the colon... :anguished: